conf/app.routes

```diff
@@ -31,7 +31,7 @@ GET         /managed-properties/:agentCode
 GET         /manage-agents                                                                    @controllers.Dashboard.manageAgents()
 
 GET         /client-properties/revoke/:plSubmissionId                                         @controllers.agent.RepresentationController.revokeClient(plSubmissionId: String)
-GET         /client-properties/revoke-client-property/:plSubmissionId/confirmed               @controllers.agent.RepresentationController.revokeClientPropertyConfirmed(plSubmissionId: String)
+POST        /client-properties/revoke-client-property/:plSubmissionId/confirmed               @controllers.agent.RepresentationController.revokeClientPropertyConfirmed(plSubmissionId: String)
```

Comment:
should it be a PATCH?

Reply:
why not POST?

Reply:
Maybe we don't care about that on the frontend?

Reply:
No we don't. It's okay.